rfc/20/README.md

```diff
@@ -115,10 +115,11 @@ TBD
 
 #### post_install
 ```
-setup_hare post_install --config <str>
+hare_setup.py --post_install
```

Comment:
Now `hare_setup.py` is obsolete. Now it is indeed `hare_setup` executable.

Reply:
Done